rllib/tuned_examples/apex_dqn/pong-apex-dqn.yaml

```diff
@@ -18,5 +18,8 @@ pong-apex:
         replay_buffer_config:
           type: MultiAgentPrioritizedReplayBuffer
           capacity: 1000000
+        # we should set this because few machines would be able to contain
+        # the replay buffers in memory otherwise
+        compress_observations: True
```

Comment:
doesn't this change a lot of things?
we will be spending tons of CPU cycles on compression and decompression right?

Reply:
removed this and opened a separate pr